assets/scss/_custom.scss

```diff
@@ -317,21 +317,64 @@ footer {
 /* DOCS */
 
 .launch-cards {
-  button {
-    cursor: pointer;
-    box-sizing: border-box;
-    background: none;
-    margin: 0;
-    border: 0;
-  }
+  padding: 0;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  row-gap: 1em;
+  .launch-card {
+    display: flex;
+    padding: 0 30px 0 0;
+    .card-content{
+      width: fit-content;
+      display: flex;
+      flex-direction: column;
+      margin: 0;
+      row-gap: 1em;
+      h2 {
+        font-size: 1.75em;
+        padding: 0.5em 0;
+        margin: 0;
+        a {
+          display: none;
+        }
+      }
+  
+      p {
+        margin: 0;
+      }
+  
+      ul {
+        list-style: none;
+        height: fit-content;
+        line-height: 1.6;
+        padding: 0;
+        margin-block-end: auto;
+      }
+  
+      br {
+        display: none;
+      }
+  
+      button {
+        height: min-content;
+        width: auto;
```

Comment:
A bit of CSS `calc()` here could improve the look further. If you want to put the work in there (it's fine not to!)

Reply:
Sure, Might as well try that. If I'll like what I see, A PR will be on it's way